README.md

```diff
@@ -51,7 +51,7 @@ may fix the problem.
 
 ``` shell
 $ docker pull paratools/salt-dev:latest
-$ docker run -it --pull --tmpfs=/dev/shm:rw,nosuid,nodev,exec \
+$ docker run -it --pull always --tmpfs=/dev/shm:rw,nosuid,nodev,exec \
```

Comment:
@wspear what was the motivation for changing this?

Reply:
At least as of docker 25.0.3 --pull requires an argument or else you get: `invalid pull option: '--tmpfs=/dev/shm:rw,nosuid,nodev,exec': must be one of "always", "missing" or "never".`